auth-web/src/views/auth/RefundView.vue

```diff
@@ -0,0 +1,303 @@
+<template>
+  <!-- TODO: Note this code is very rough and will need to be rewritten. -->
+  <v-app>
+    <v-container>
+      <h2 class="mb-4">
+        MVP Refunds - OPS only for now.
+      </h2>
+      <v-text-field
+        v-model="invoiceId"
+        label="Invoice Number EG. (REG00012343 or 12343)"
+      />
+      <v-btn
+        :disabled="!invoiceId.trim()"
+        @click="fetchInvoice"
+      >
+        Fetch Invoice
+      </v-btn>
+      <v-radio-group
+        v-model="refundType"
+        row
+      >
+        <v-radio
+          label="Full refund"
+          :value="RefundType.FULL"
+        />
+        <v-radio
+          label="Partial Refund"
+          :value="RefundType.PARTIAL"
+        />
+      </v-radio-group>
+      <v-simple-table>
+        <thead>
+          <tr>
+            <th id="line-item-number-header">
+              Line Item #
+            </th>
+            <th id="description-header">
+              Description
+            </th>
+            <th id="filing-fee-header">
+              Filing Fee
+            </th>
+            <th id="priority-fee-header">
+              Priority Fees
+            </th>
+            <th id="future-effective-fee-header">
+              Future Effective Fees
+            </th>
+            <th id="service-fee-header">
+              Service Fees
+            </th>
+            <th
+              v-if="refundType === RefundType.PARTIAL"
+              id="action-header"
+            >
+              Actions
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr
+            v-for="(item, index) in paymentLineItems"
+            :key="index"
+          >
+            <td>{{ index + 1 }}</td>
+            <td>{{ item.description }}</td>
+            <td>
+              <v-text-field
```

Comment:
Only the total is wired up right now I think

Reply:
<img width="1415" alt="image" src="https://github.com/bcgov/sbc-auth/assets/16507223/790cff22-eb46-4c15-8736-f9ec91b1fb90">
it calculate total for each, but send them separately by LineItemId on payload